### DISCUSSION TITLE
ICF Disgruntlement

### DISCUSSION THREAD

**Discussion by m0therforker** [gaiaus/ICF#35] [General]
![Screenshot_20231101-210952.png](https://github.com/gaiaus/ICF/assets/142172797/c83ee123-8b38-4052-9dd3-fab2586bce97)

![Screenshot_20231102-055757.png](https://github.com/gaiaus/ICF/assets/142172797/6690d164-e678-444d-8081-e67f467582e6)

![Screenshot_20231102-055821.png](https://github.com/gaiaus/ICF/assets/142172797/d9a0f43c-8abf-4845-9145-b27c8d20172a)

![Screenshot_20231102-055847.png](https://github.com/gaiaus/ICF/assets/142172797/5b4cb9fa-c748-4aea-a308-91e52aa4f580)

![Screenshot_20231102-060110.png](https://github.com/gaiaus/ICF/assets/142172797/31ccdd6f-4fae-49bb-a85b-dfa361c2355e)

![Screenshot_20231102-060214.png](https://github.com/gaiaus/ICF/assets/142172797/b28d1799-d946-4b23-8b44-bb3e9d341441)

![Screenshot_20231102-060233.png](https://github.com/gaiaus/ICF/assets/142172797/1d9c9f04-73b6-4db4-b481-22791351c40d)

![Screenshot_20231102-060459.png](https://github.com/gaiaus/ICF/assets/142172797/7b39ca8c-9539-4080-a11e-c9f4c2939129)

![Screenshot_20231102-060643.png](https://github.com/gaiaus/ICF/assets/142172797/fbb3390b-3757-4f2d-b5f5-8a2b4db92bd0)

![Screenshot_20231102-061026.png](https://github.com/gaiaus/ICF/assets/142172797/479ea9f3-bf15-448e-b1fb-269e449fea04)

![Screenshot_20231102-061411.png](https://github.com/gaiaus/ICF/assets/142172797/1d989fd6-4993-440b-b5cc-7e560b1cdbfb)

![Screenshot_20231102-061357.png](https://github.com/gaiaus/ICF/assets/142172797/ee30799f-5e5e-4dab-ba94-ece7edbc3911)

![Screenshot_20231102-061314.png](https://github.com/gaiaus/ICF/assets/142172797/0f6b7082-891f-4360-8dd1-36e1c602712b)

![Screenshot_20231102-061248.png](https://github.com/gaiaus/ICF/assets/142172797/04f0af5e-14e5-41bf-a9d0-efd0e1393113)

![Screenshot_20231102-061233.png](https://github.com/gaiaus/ICF/assets/142172797/26c6fad0-fa35-453e-b21b-0f1feaccc075)

![Screenshot_20231102-061248.png](https://github.com/gaiaus/ICF/assets/142172797/f8a5e588-0545-4934-bedb-639d5ddc8e8f)

![Screenshot_20231102-061144.png](https://github.com/gaiaus/ICF/assets/142172797/fdbc2bed-bd7d-49cb-8068-2816f3405394)

![Screenshot_20231102-061640.png](https://github.com/gaiaus/ICF/assets/142172797/9a305974-1808-446e-b25f-5128f0114dc3)

![Screenshot_20231102-061659.png](https://github.com/gaiaus/ICF/assets/142172797/b15e2281-6e20-439c-84b8-3018ecaedcc9)

![Screenshot_20231102-061712.png](https://github.com/gaiaus/ICF/assets/142172797/1a6d5e9e-51bd-497f-9566-2b489f8b335f)

![Screenshot_20231102-061749.png](https://github.com/gaiaus/ICF/assets/142172797/863676ac-bf15-4776-9322-ef79b6d81010)

![Screenshot_20231102-061916.png](https://github.com/gaiaus/ICF/assets/142172797/8621180a-8462-4bda-8ed4-91c5efeaaa0c)

![Screenshot_20231102-061941.png](https://github.com/gaiaus/ICF/assets/142172797/7a710650-9b79-4bf2-ad27-f1062c773608)

![Screenshot_20231102-062002.png](https://github.com/gaiaus/ICF/assets/142172797/c23cdabb-539a-451c-af87-2556c2fc94bb)

![Screenshot_20231102-062026.png](https://github.com/gaiaus/ICF/assets/142172797/de5e3b32-27f7-49fe-bffd-2e1d6e88c2db)

![Screenshot_20231102-062039.png](https://github.com/gaiaus/ICF/assets/142172797/f5a45393-a7fc-480a-a1b5-87d174f8ffb4)

![Screenshot_20231102-062513.png](https://github.com/gaiaus/ICF/assets/142172797/9700d212-4103-41d8-aa4a-629b47a05b64)

![Screenshot_20231102-062455.png](https://github.com/gaiaus/ICF/assets/142172797/de4536e0-0229-4421-bfd7-0360cbd8b4a0)

![Screenshot_20231102-062438.png](https://github.com/gaiaus/ICF/assets/142172797/66eb452c-d0b1-4a11-be7f-b36dae30d7f7)

![Screenshot_20231102-062408.png](https://github.com/gaiaus/ICF/assets/142172797/5ca28c43-3106-4c5e-bf30-ab3554cb5ff1)

![Screenshot_20231102-062312.png](https://github.com/gaiaus/ICF/assets/142172797/595a6fe2-4a9d-47e6-b07a-bb055308bace)

![Screenshot_20231102-065457.png](https://github.com/gaiaus/ICF/assets/142172797/e0c25120-1aff-40b5-b073-193adbc99d23)

![Screenshot_20231102-065522.png](https://github.com/gaiaus/ICF/assets/142172797/4a675e4b-b60d-4a42-8d7d-81bf27873991)

![Screenshot_20231102-081633.png](https://github.com/gaiaus/ICF/assets/142172797/1afa6f58-5c5b-4321-849d-29cb53386aef)

![Screenshot_20231102-081711.png](https://github.com/gaiaus/ICF/assets/142172797/510611b5-bdd0-4f7a-85b7-5b2d2c205916)

![Screenshot_20231102-082238.png](https://github.com/gaiaus/ICF/assets/142172797/337b2ae4-f58c-44fb-83e6-30b64255d4ee)

![Screenshot_20231102-082344.png](https://github.com/gaiaus/ICF/assets/142172797/1d065d7b-4d99-45ac-bad3-0e2c3b01839e)

![Screenshot_20231102-082445.png](https://github.com/gaiaus/ICF/assets/142172797/4990c14b-4eaa-4c54-9eb6-99fe2401306b)

![Screenshot_20231102-083025.png](https://github.com/gaiaus/ICF/assets/142172797/1a678d7c-c3e5-450a-88a4-a3c9af32b101)

![Screenshot_20231102-082948.png](https://github.com/gaiaus/ICF/assets/142172797/dc0a9dae-7e33-43dc-976d-fcb14db37138)

![Screenshot_20231102-082929.png](https://github.com/gaiaus/ICF/assets/142172797/8e306c7d-7f6a-45b1-944e-c2c1b45cd955)

![Screenshot_20231102-082857.png](https://github.com/gaiaus/ICF/assets/142172797/19e90c28-f4ea-419e-9450-c552c6acb5aa)

![Screenshot_20231102-082828.png](https://github.com/gaiaus/ICF/assets/142172797/bba1be22-343e-42ac-9ae6-90b2499add96)

![Screenshot_20231102-082857.png](https://github.com/gaiaus/ICF/assets/142172797/b147bd1c-1d96-411c-b60f-a4ccd4aef633)

![Screenshot_20231102-082814.png](https://github.com/gaiaus/ICF/assets/142172797/34604280-e9e5-46d1-b682-c883be2bb9fc)

![Screenshot_20231102-082756.png](https://github.com/gaiaus/ICF/assets/142172797/89768dca-179a-4d2a-9037-4612fc14b3c9)

![Screenshot_20231102-082857.png](https://github.com/gaiaus/ICF/assets/142172797/10094633-c0cd-41f5-a912-305375a9401e)

![Screenshot_20231102-082948.png](https://github.com/gaiaus/ICF/assets/142172797/5f061499-10ac-4475-9b2a-8ebab6ee2350)

![Screenshot_20231102-082929.png](https://github.com/gaiaus/ICF/assets/142172797/0672ff17-b3fe-40e2-8848-d5c33f8f87b5)

![Screenshot_20231102-083436.png](https://github.com/gaiaus/ICF/assets/142172797/3766d523-39f7-4811-b37f-32c217682054)

![Screenshot_20231102-083025.png](https://github.com/gaiaus/ICF/assets/142172797/9b42919b-5636-4b07-9de0-4ca304f5d8d6)

![Screenshot_20231102-083556.png](https://github.com/gaiaus/ICF/assets/142172797/2279ebb0-f80a-497c-bb6a-30270161f8ea)

![Screenshot_20231102-083459.png](https://github.com/gaiaus/ICF/assets/142172797/6dc7429d-24ab-4951-b463-43879a5aaf11)

![Screenshot_20231102-083707.png](https://github.com/gaiaus/ICF/assets/142172797/d747fa17-0b97-4d65-beb2-6a459ae5de72)

![Screenshot_20231102-083629.png](https://github.com/gaiaus/ICF/assets/142172797/003c2974-2cae-4174-bb52-328bd37008d4)

![Screenshot_20231102-083836.png](https://github.com/gaiaus/ICF/assets/142172797/ed7148ef-e21a-4bc8-8d12-d1bcb22cf130)

![Screenshot_20231102-083740.png](https://github.com/gaiaus/ICF/assets/142172797/fdee9583-892e-48f9-8423-7ec9eb549515)

![Screenshot_20231102-084318.png](https://github.com/gaiaus/ICF/assets/142172797/6f668b78-9cb9-4f59-9229-3db7522b68e9)

![Screenshot_20231102-084330.png](https://github.com/gaiaus/ICF/assets/142172797/5679d6ec-2cc9-4bb6-902e-e06035061872)

![Screenshot_20231102-085325.png](https://github.com/gaiaus/ICF/assets/142172797/296d5080-9fe0-4fa4-a529-591e952516d9)

![Screenshot_20231102-085406.png](https://github.com/gaiaus/ICF/assets/142172797/e389ec89-597a-4053-907e-6ce7e13f8ecf)

![Screenshot_20231102-085454.png](https://github.com/gaiaus/ICF/assets/142172797/55a44d6d-2a5a-43c8-982e-b76d25430d43)

![Screenshot_20231102-085518.png](https://github.com/gaiaus/ICF/assets/142172797/6c2a5750-6378-4140-ba22-5054cc0d5079)

![Screenshot_20231102-085614.png](https://github.com/gaiaus/ICF/assets/142172797/fe33d58f-7a4d-4e38-8f02-fe67bac1cbc2)

![Screenshot_20231102-085649.png](https://github.com/gaiaus/ICF/assets/142172797/734c14ac-5d38-46e7-91e2-80ab27c32a8f)

![Screenshot_20231102-085707.png](https://github.com/gaiaus/ICF/assets/142172797/cd1ec6d7-42bf-4cb9-bcce-98cf1b1953c0)

![Screenshot_20231102-085732.png](https://github.com/gaiaus/ICF/assets/142172797/35ddad33-c0f7-4101-8e3d-d889c2c9696d)

![Screenshot_20231102-085749.png](https://github.com/gaiaus/ICF/assets/142172797/39ef50b7-60ff-42f8-a8ea-f9984ab649e6)

![Screenshot_20231102-085816.png](https://github.com/gaiaus/ICF/assets/142172797/ec0edc95-f094-4d23-a65d-78520f075f0c)

![Screenshot_20231102-085913.png](https://github.com/gaiaus/ICF/assets/142172797/fefba891-4cd3-4f6c-9c1f-ee72c943bd5b)

![Screenshot_20231102-085944.png](https://github.com/gaiaus/ICF/assets/142172797/bc0cee41-2a43-41ad-9e13-660aa8677c00)

![Screenshot_20231102-090002.png](https://github.com/gaiaus/ICF/assets/142172797/80138aab-0468-4776-83e8-20448f6b7bf7)

![Screenshot_20231102-090019.png](https://github.com/gaiaus/ICF/assets/142172797/c952ad3b-cc5f-4f45-aa24-3ffe7c19e5bc)

![Screenshot_20231102-092906.png](https://github.com/gaiaus/ICF/assets/142172797/c06aabfd-debf-4b0c-a9eb-62cbdb0ad1f2)

![Screenshot_20231102-093107.png](https://github.com/gaiaus/ICF/assets/142172797/f679ae0e-440c-445b-a0d7-c8c6869438cd)

![Screenshot_20231103-010934.png](https://github.com/gaiaus/ICF/assets/142172797/fbd7a53b-6452-4516-806e-b51bc1e050fe)

![Screenshot_20231103-011818.png](https://github.com/gaiaus/ICF/assets/142172797/9c008acd-1591-46c5-b80b-93ef6ff4cd80)

![Screenshot_20231103-023829.png](https://github.com/gaiaus/ICF/assets/142172797/ef20aba9-7f69-47e2-9812-b06dbead7823)

**Comment by m0therforker** [comment]
https://twitter.com/gyunit_/status/1717739964807397482?t=d8YFEsj2u7zM8GaQzOoi0A&s=19

**Comment by gaiaus** [comment]
![Screenshot_20231106-231755.png](https://github.com/gaiaus/ICF/assets/146326929/6b8ced02-cc73-4910-a91d-2ebd4444d606)

![Screenshot_20231106-231809.png](https://github.com/gaiaus/ICF/assets/146326929/09627be9-d03c-4075-b7e3-cfae9856dbc3)

**Comment by gaiaus** [comment]
![Screenshot_20231107-045834.png](https://github.com/gaiaus/ICF/assets/146326929/48acf830-ff90-451d-a0c7-18104116a586)
